src/main/scala/com/yotpo/metorikku/configuration/metric/DQCheckDefinition.scala

```diff
@@ -0,0 +1,20 @@
+package com.yotpo.metorikku.configuration.metric
+
+import com.fasterxml.jackson.core.`type`.TypeReference
+import com.fasterxml.jackson.module.scala.JsonScalaEnumeration
+
+case class DQCheckDefinition(@JsonScalaEnumeration(classOf[DQCheckOpTypeReference]) op: DQCheckOpType.Op,
+                             column: Option[String],
+                             values: List[Option[Any]],
+                             level: Option[String]) extends Serializable
+case class DQCheckDefinitionList(checks: List[DQCheckDefinition],
+                                 level: Option[String]) extends Serializable
```

Comment:
every check has a level, what is the meaning of a level at the list scope?

Reply:
You can pass level for a single check or for the entire step.
the check level overrides the step level